bin/travis/before_script.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+"$(dirname ${BASH_SOURCE[0]})/../../node_modules/.bin/sequelize" \
+  --config /etc/blocktogether/sequelize.json db:migrate
+"$(dirname ${BASH_SOURCE[0]})/../../run-dev.sh" \
+  > "$(dirname ${BASH_SOURCE[0]})/../../bt.log" 2>&1 &
```

Comment:
Why redirect this to a file? If you let it go to stdout, people can read it while waiting for the test to complete.